experimental/rest/api.go

```diff
@@ -0,0 +1,83 @@
+package rest
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+// Client interface
+type Client interface {
+	Fetch(ctx context.Context, uriPath string, uriQuery string) (*http.Response, error)
+}
```

Comment:
Shall we provide an implementation for this interface in the SDK?

Reply:
the implementation in appd had some [appd specific stuff.](https://github.com/grafana/appdynamics-grafana-datasource/pull/314/files#diff-34feea038c72e57c85d266e02fc46dbd38ae9e57a9c16b90f418faaf1d11ce07L61)  but yes that would be great